packages/dx-react-grid-bootstrap3/src/templates/table-group-cell/cell.jsx

```diff
@@ -17,19 +19,21 @@ export const Cell = ({
     onClick={onToggle}
     {...restProps}
   >
-    <Icon
-      expanded={expanded}
-      onToggle={onToggle}
-      style={{
-        marginRight: '8px',
-      }}
-    />
-    <Content
-      column={column}
-      row={row}
-    >
-      {children}
-    </Content>
+    <Container style={{ left }}>
+      <Icon
+        expanded={expanded}
+        onToggle={onToggle}
+        style={{
```

Comment:
Property value is new on every `render` thought it actually never changes. Extract a constant.